packages/calypso-e2e/src/lib/components/preview-component.ts

```diff
@@ -3,6 +3,9 @@ import { BaseContainer } from '../base-container';
 
 const selectors = {
 	previewPane: '.web-preview',
+
+	// Actions on pane
+	closeButton: 'button:text("Close")',
```

Comment:
Definitely optional suggestion here...

Would using `button[aria-label='Close preview']` potentially be more resilient here than the text selector?

My only wondering is if in future someone adds a button to the page behind this preview that says something like "Close xyz", it would hijack this selector. 

Aria-labels carry their own risk too though... What do you think?

Reply:
Good question!

[Playwright docs](https://playwright.dev/docs/selectors/#prioritize-user-facing-attributes) recommends _user-facing attributes should be prioritized_ and includes both text and aria attributes in the example.

You raise an excellent point about another `Close xyz` button on the page behind the PreviewComponent.
It wouldn't be an issue if we prefixed the selector with the top-level CSS selector for the PreviewComponent, but that would be violating the best practice.

I'll make your suggested changes in the next push :)